### DISCUSSION TITLE
Community Meeting 2022-09-07

### DISCUSSION THREAD

**Discussion by JamieMagee** [microsoft/component-detection#195] [Announcements]
📆 Wednesday 7th September 2022
🕙 10AM Pacific Time ([Find in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20220907T17))
☎️ [Teams meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_YmVmMjcxMmEtMmYwYi00MGNiLTg4MDgtODNiMDY5ZjNkYjY3%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22bb41e436-5c77-44e2-b783-1ad2b9f7aa7f%22%7d)

## Agenda

- Upcoming breaking changes #240, #251, and #107 

## Attendees

- @cobya (Microsoft)
- @Omotola (Microsoft)
- @grvillic (Microsoft)
- @amitla1 (Microsoft)
- Adrian Diglio (Microsoft)

## Recording

https://user-images.githubusercontent.com/1358764/194116552-464316e0-999c-4287-97b9-02bde4b9f774.mp4

**Comment by JamieMagee** [comment]
- [Issues opened since last meeting](https://github.com/microsoft/component-detection/issues?q=is%3Aissue+created%3A%3E2022-08-10)
- [PRs opened since last meeting](https://github.com/microsoft/component-detection/pulls?q=is%3Apr+created%3A%3E2022-08-10)

**Comment by cobya** [comment]
Recording has been published in the post above. Thanks for attending everyone!
